src/components/Button.js

```diff
@@ -0,0 +1,20 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+
+Button.propTypes = {
+  onClick: PropTypes.func.isRequired,
+  isActive: PropTypes.bool,
+  children: PropTypes.node,
+}
+
+export default function Button(props) {
```

Comment:
You can export just the styled.button, that's cleaner and you don't have to check for the props